Vagrantfile

```diff
@@ -219,8 +219,8 @@ def install_pythons(boxname)
     pyenv install 3.6.2  # tests
     pyenv install 3.7.11  # binary build, use latest 3.7.x release
     pyenv install 3.8.0  # tests
-    pyenv install 3.9.0  # tests
-    pyenv install 3.10-dev  # tests
+    pyenv install 3.9.9  # tests
```

Comment:
no, this should stay at 3.9.0.

we usually test at the oldest patch-release that works (usually .0).
the only exception is the python release for which we build binaries (and thus do not use .0, but .latest).

Reply:
in this case, we use 3.7.11 for the binaries.